cli/util/osfs.go

```diff
@@ -0,0 +1,22 @@
+package util
+
+import (
+	"io/fs"
+	"os"
+)
+
+type osFS struct{}
+
+var osfs osFS
+
+func GetOsFS() fs.FS {
+	return osfs
+}
```

Comment:
The global variable is confusing here. Please, add a comments for the type.
```suggestion

func GetOsFS() fs.FS {
	return osFS{}
}
```

Reply:
I don't think there are enough allocations here for us to worry about creating the object.

The global variable is more confusing - it suggests the idea of ​​replacing the global context in tests.

Reply:
OK, fixed as suggested.